xclim/indices/run_length.py

```diff
@@ -388,6 +388,46 @@ def run_end_after_date(
     return end.where(beg.notnull())
 
 
+def first_run_after_date(
+    da: xr.DataArray,
+    window: int,
+    date: str = "07-01",
+    dim: str = "time",
+    coord: str = "dayofyear",
```

Comment:
```suggestion
    coord: Opitonal[str] = "dayofyear",
```

Reply:
haha you sure?

Reply:
It's marked optional in the docstring. Both can't be right?

Reply:
oui oui you're right, I meant for the "opitonal" ;)

Reply:
Oof. That's embarrassing.